imperial_coldfront_plugin/models.py

```diff
@@ -29,3 +29,24 @@ class GroupMembership(models.Model):
         on_delete=models.CASCADE,
         related_name="group_memberships_set",
     )
+
+
+class UserID(models.Model):
+    """Identity data to map a user to a unique identifier.
+
+    This model stores a unique identifier for each user, which can be used to
+    identify users in external systems. The identifier is an integer value that
+    is unique to each user user.
+
+    Attributes:
+        user (OneToOneField): A reference to the user, connected to AUTH_USER_MODEL.
+            Deletes the related identifier when the user is deleted.
+        identifier (IntegerField): A unique identifier for the user.
+    """
+
+    user = models.OneToOneField(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="user_id",
+    )
+    identifier = models.IntegerField(default=0)
```

Comment:
Lets remove the default value here as we'll always need to provide it. Where uid's are concerned 0 is a special value that refers to the root user (or superadmin).

Reply:
I see. Done!